packages/blocks/src/database-block/utils/database-init.ts

```diff
@@ -0,0 +1,43 @@
+import {
+  assertExists,
+  type BaseBlockModel,
+  type Page,
+} from '@blocksuite/store';
+
+export function initDatabaseBlock(
```

Comment:
This looks better to be defined as a method in BlockService

Reply:
ok,  i will change it.